### ISSUE TITLE
[Reimplementation] no file:sodad_train.json

### ISSUE THREAD

**Issue by Polarisamoon** [shaunyuan22/SODA-mmdetection#19]
### Prerequisite

- [X] I have searched [Issues](https://github.com/open-mmlab/mmdetection/issues) and [Discussions](https://github.com/open-mmlab/mmdetection/discussions) but cannot get the expected help.
- [X] I have read the [FAQ documentation](https://mmdetection.readthedocs.io/en/latest/faq.html) but cannot get the expected help.
- [X] The bug has not been fixed in the [latest version (master)](https://github.com/open-mmlab/mmdetection) or [latest version (3.x)](https://github.com/open-mmlab/mmdetection/tree/dev-3.x).

### 💬 Describe the reimplementation questions

When I preprocess the SODA-D data，use：python tools/img_split/sodad_split.py --cfgJson sodad_train.json ，found no such file ：sodad_train.json，but when run：python tools/img_split/sodad_split.py，it works，but only create train.json and images，no test and val.json，should i change split_train.json to obtain test.json and val.json?

### Environment

mmdet2.26
mmcv1.7.1

**Comment by Polarisamoon** [comment]
should i change the "mode" in split_train.json to "val "or "test" to obtain val and test data? Could you please take some time out of your busy schedule to answer my question

**Comment by shaunyuan22** [comment]
`python tools/img_split/sodad_split.py --cfgJson ./tools/img_split/split_configs/split_train.json` works. Make sure you used the correct path. You should create `split_val.json` and `split_test.json` and change the corresponding paths to obtain divided files for validation and testing.

**Comment by Polarisamoon** [comment]
> `python tools/img_split/sodad_split.py --cfgJson ./tools/img_split/split_configs/split_train.json` works. Make sure you used the correct path. You should create `split_val.json` and `split_test.json` and change the corresponding paths to obtain divided files for validation and testing.

Thank you for your response. My validation set has 109,126 images, and the test set has 161,579 images, far more than the 58,023 images in the training set. Is that correct?

**Comment by shaunyuan22** [comment]
right, since we cannot filter empty patches for validation and testing
